idl/github.com/uber/cadence/shared.thrift

```diff
@@ -365,10 +365,15 @@ struct Decision {
 
 struct WorkflowExecutionStartedEventAttributes {
   10: optional WorkflowType workflowType
+  12: optional string parentWorkflowDomain
+  14: optional WorkflowExecution parentWorkflowExecution
+  16: optional i64 (js.type = "Long") parentInitiatedEventId
   20: optional TaskList taskList
   30: optional binary input
   40: optional i32 executionStartToCloseTimeoutSeconds
   50: optional i32 taskStartToCloseTimeoutSeconds
+  52: optional ChildPolicy childPolicy
```

Comment:
if the "child policy" represents what this workflow's parent want this workflow to do, then we should use a better name

Reply:
We are keeping our API as close to SWF as possible: https://docs.aws.amazon.com/amazonswf/latest/apireference/API_WorkflowExecutionStartedEventAttributes.html